test_runner/regress/test_tenant_delete.py

```diff
@@ -539,8 +539,10 @@ def hit_run_failpoint():
 
         # Permit the duplicate background request to run to completion and fail.
         ps_http.configure_failpoints((BEFORE_RUN_FAILPOINT, "off"))
-        with pytest.raises(PageserverApiException, match=CONFLICT_MESSAGE):
-            background_4xx_req.result(timeout=10)
+        background_4xx_req.result(timeout=10)
```

Comment:
nit: `background_4xx_req` is probably the wrong name with the changes in this PR

Reply:
You're right, but I expect this test will go away (along with much of DeleteTenantFlow) when we implement simpler deletion, so not very invested in cleaning  up -- going to merge this now to de-flake the test.